cabal-testsuite/src/Test/Cabal/Prelude.hs

```diff
@@ -544,7 +544,6 @@ src `archiveTo` dst = do
     -- TODO: --format ustar, like createArchive?
     -- --force-local is necessary for handling colons in Windows paths.
     tar $ ["-czf", dst]
-       ++ ["--force-local" | buildOS == Windows]
```

Comment:
For the reviewer: I didn't require this, but perhaps because I maybe have modified my `tar`. If it fails lets make this dependent on whether we are using bsdtar or not.